packages/gp2-components/src/organisms/FilterPills.tsx

```diff
@@ -0,0 +1,39 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { pixels } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import FilterPill from '../molecules/FilterPill';
+
+const { rem } = pixels;
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  flexWrap: 'wrap',
+  width: '100%',
+  gap: rem(8),
+  paddingTop: rem(24),
+});
+
+type FiltersType = keyof Pick<
+  gp2Model.FetchUsersFilter,
+  'keywords' | 'regions' | 'workingGroups' | 'projects'
+>;
+type FilterPillsProps = {
+  pills: { typeOfFilter: FiltersType; label: string; id: string }[];
```

Comment:
maybe `type` or `filterType`

Reply:
I don't like the name pills here, since its not the pills yet, it's more of a filterList or appliedFiltersList